rust/headless-client/src/dns_control/windows.rs

```diff
@@ -46,20 +43,19 @@ impl DnsController {
     ///
     /// Must be async to match the Linux signature
     #[allow(clippy::unused_async)]
-    pub(crate) async fn set_dns(&mut self, dns_config: &[IpAddr]) -> Result<()> {
-        deactivate().context("Failed to deactivate DNS control")?;
+    pub(super) async fn set_dns(&mut self, dns_config: &[IpAddr]) -> Result<()> {
         activate(dns_config).context("Failed to activate DNS control")?;
         Ok(())
     }
 
     /// Flush Windows' system-wide DNS cache
     ///
     /// `&self` is needed to match the Linux signature
-    pub(crate) fn flush(&self) -> Result<()> {
+    pub(super) fn flush(&self) -> Result<()> {
         tracing::debug!("Flushing Windows DNS cache...");
-        Command::new("powershell")
+        Command::new("ipconfig")
```

Comment:
Powershell seems heavy, like Python, it just takes 300-500 ms to start up and do nothing. `ipconfig` is older but much faster.